### PR TITLE
add limit range

### PR DESCRIPTION
**Pull request by zouyee**
add limit range metrics and readme,thanks
@brancz

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kube-state-metrics/114)

### CODE REVIEW

**Comment by brancz** [comment]
lgtm :+1:

File: limitrange.go [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+// Describe implements the prometheus.Collector interface.
+func (rqc *limitRangeCollector) Describe(ch chan<- *prometheus.Desc) {
+	ch <- descLimitRange
+}
+
+// Collect implements the prometheus.Collector interface.
+func (rqc *limitRangeCollector) Collect(ch chan<- prometheus.Metric) {
+	limitRangeCollector, err := rqc.store.List()
+	if err != nil {
+		glog.Errorf("listing resource quotas failed: %s", err)
```
Comment (andyxning) [inline limitrange.go]: @zouyee @brancz 
`s/"reource quotas"/"limit range"/g`

File: limitrange.go [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+		return
+	}
+
+	for _, rq := range limitRangeCollector.Items {
+		rqc.collectLimitRange(ch, rq)
+	}
+}
+
+func (rqc *limitRangeCollector) collectLimitRange(ch chan<- prometheus.Metric, rq v1.LimitRange) {
```
Comment (andyxning) [inline limitrange.go]: `s/rpc/lrc/g`

**Comment by brancz** [comment]
@andyxning feel free to open a PR if you want to fix those issues :)

**Pull request merged**